src/main/java/github/scarsz/discordsrv/objects/threads/PresenceUpdater.java

```diff
@@ -77,27 +80,44 @@ public void run() {
                     status = PlaceholderUtil.replacePlaceholders(status);
                     status = MessageUtil.strip(status); // remove color codes
                 }
-                boolean same = Objects.equals(lastStatus, status);
+
+                String onlineStatusString = DiscordSRV.config().getString("DiscordOnlineStatus").toUpperCase(Locale.ROOT).trim();
+                OnlineStatus onlineStatus = OnlineStatus.ONLINE;
+
+                switch (onlineStatusString) {
```

Comment:
No reason for this switch to exist when the configured values are identical to the enum values, Enum#valueOf(name) would be preferred

Having the configured values be the same isn't good either though. Should be something like `dnd` or at least get rid of the `_`'s with something like `do not disturb`